.dockerignore

```diff
@@ -0,0 +1,5 @@
+**/.git
+**/.gitignore
+**/.gitmodules
+.dockerignore
+Dockerfile
```

Comment:
Should Dockerfile really be ignored? That seems counter-intuitive..

Reply:
It is a matter of taste.

`.dockerignore` affects `COPY` command from `Dockerfile`. Basically it means that docker will not copy `.dockerignore` and `Dockerfile` to the build image.

Some people prefer to have `Dockerfile` inside an image just for documentation purposes. But due to two-stage build, it will not affect the final image anyway.

Reply:
Oh, I realize why it made no sense to me now. I thought this was a `.gitignore` file, probably read the filename from the file or something. 🤦‍♂️ 
Nothing wrong with this at all then 👍